src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const mainImage = document.querySelector('#largeImg');
+
+  const thumbnails = document.querySelector('#thumbs');
+
+  function changeImage(e) {
+    const necessaryLink = e.target.closest('a');
```

Comment:
The variable name 'necessaryLink' is not very descriptive. Consider using a more descriptive name, for example 'clickedLink', to make the code more readable.